mindsdb_native/libs/data_sources/mongodb_ds.py

```diff
@@ -8,20 +8,28 @@
 from mindsdb_native.libs.data_types.data_source import DataSource
 
 class MongoDS(DataSource):
-    def __init__(self, query, collection, database='database',
+    def __init__(self, query, collection,
+                 database='database', limit=None, sort_by=None,
                  host='127.0.0.1', port=None, user=None, password=None):
```

Comment:
don't change argument order, add the new ones at the end.

Reply:
Fixed